pkg/cmdlets/auth/login.go

```diff
@@ -68,6 +68,23 @@ func AuthenticateWithInput(username, password, totp string, cfg keypairs.Configu
 	return nil
 }
 
+// AuthenticateWithInput will prompt the user for authentication if the input doesn't already provide it
```

Comment:
Should be AuthenticateWithToken. Comment body should probably also be updated to mention token instead of input.